### DISCUSSION TITLE
Match with humanizing mutations

### DISCUSSION THREAD

**Discussion by salvatta2** [brianhie/efficient-evolution#16] [Q&A]
Dears,

impressive model and congrats for the recentlly published paper! I have run a bunch of mouse Ab and I have realised that some of the highly recommended mutations are mutations to human germline residues (for the corresponding V family). As far as I understand (but maybe I am wrong), the model is trained with protein sequences that cover a wide range of species. Why is then evolutionary fitness correlated with "humanizing" mutations?

Kind regards,
Salva

**Comment by brianhie** [comment]
Interesting, I would be interested in quantifying if this is statistically significant compared to other cases. For example, human to mouse germline or involving other species.
